### PR TITLE
Update Shunt Compensator form

### PR DESCRIPTION
**Pull request by ayolab**

### CODE REVIEW

File: src/translations/fr.json [hunk 1]
```diff
@@ -522,6 +527,7 @@
     "ShuntCompensatorErrorMaximumLessThanOne": "Le nombre maximum de sections doit être supérieur à 0",
     "ShuntCompensatorErrorCurrentLessThanZero": "Le nombre de sections utilisées doit être supérieur ou égal à 0",
     "ShuntCompensatorErrorCurrentLessThanMaximum": "Le nombre de sections utilisées doit être inférieur ou égal au nombre maximum",
+    "ShuntCompensatorErrorQAtNominalVoltageLessThanZero": "La valeur de Q à Unom doit être supérieure ou égale à 0",
```
Comment (FranckLecuyer) [inline src/translations/fr.json:530]: Missing in en.json

File: src/components/refactor/rhf-inputs/radio-input.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    const intl = useIntl();
+
+    return (
+        <FormControl
+            style={{
+                marginTop: '-12px',
+            }}
+        >
+            {label && (
+                <FormLabel id={id ? id : label}>
```
Comment (FranckLecuyer) [inline src/components/refactor/rhf-inputs/radio-input.js:20]: id ?? label

File: src/components/refactor/rhf-inputs/radio-input.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+                    <FormattedMessage id={label} />
+                </FormLabel>
+            )}
+            <RadioGroup
+                row
+                aria-labelledby={id ? id : label}
```
Comment (FranckLecuyer) [inline src/components/refactor/rhf-inputs/radio-input.js:26]: id ?? label

File: src/components/refactor/rhf-inputs/radio-input.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import FormControl from '@mui/material/FormControl';
```
Comment (FranckLecuyer) [inline src/components/refactor/rhf-inputs/radio-input.js:1]: Missing Copyright header

File: src/components/refactor/rhf-inputs/radio-input.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+                        value={value.id}
+                        key={value.id}
+                        label={intl.formatMessage({ id: value.label })}
+                    />
+                ))}
+            </RadioGroup>
+        </FormControl>
+    );
+};
+
```
Comment (FranckLecuyer) [inline src/components/refactor/rhf-inputs/radio-input.js:42]: Missing propTypes

File: src/components/refactor/rhf-inputs/enum-input.js [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+            )}
+        </FormControl>
+    );
+};
+
+EnumInput.propTypes = {
+    name: PropTypes.string.isRequired,
+    label: PropTypes.string.isRequired,
+    options: PropTypes.array.isRequired,
+    previousValue: PropTypes.object,
```
Comment (FranckLecuyer) [inline src/components/refactor/rhf-inputs/enum-input.js:59]: previousValue is not a prop
Missing 'size' prop

File: src/components/refactor/dialogs/shunt-compensator-creation/shunt-compensator-creation-form.js [hunk 7]
```diff
@@ -102,6 +88,27 @@ const ShuntCompensatorCreationForm = ({ studyUuid, currentNode }) => {
...
 
+    const reactiveNominalPower = (
+        <FloatInput
+            name={Q_AT_NOMINAL_V}
+            label={'QatNominalV'}
+            adornment={ReactivePowerAdornment}
+        />
+    );
+
+    const reactiveNominalPowerType = (
```
Comment (FranckLecuyer) [inline src/components/refactor/dialogs/shunt-compensator-creation/shunt-compensator-creation-form.js:99]: shuntCompensatorType ?

File: src/components/refactor/dialogs/shunt-compensator-creation/shunt-compensator-creation-dialog.js [hunk 8]
```diff
@@ -50,20 +55,43 @@ const schema = yup
...
-                'ShuntCompensatorErrorCurrentLessThanMaximum'
-            )
-            .required(),
-        [IDENTICAL_SECTIONS]: yup.bool().required(),
-        [SUSCEPTANCE_PER_SECTION]: yup.number().nullable().required(),
+            .nullable()
+            .when([CHARACTERISTICS_CHOICE], {
+                is: (reactivePowerControl) =>
+                    reactivePowerControl ===
+                    CHARACTERISTICS_CHOICES.Q_AT_NOMINAL_V.id,
```
Comment (YenguiSeddik) [inline src/components/refactor/dialogs/shunt-compensator-creation/shunt-compensator-creation-dialog.js:86]: I think here we can directly check the value of reactivePowerControl:
.when([CHARACTERISTICS_CHOICE], {
                is: CHARACTERISTICS_CHOICES.Q_AT_NOMINAL_V.id

**Review by FranckLecuyer — APPROVED** [review]

**Pull request merged**